### DISCUSSION TITLE
自动删除 ProcessInstance

### DISCUSSION THREAD

**Discussion by chinashenkai** [apache/dolphinscheduler#7602] [Q&A]
使用场景: 我们的任务调度为每分钟一次, 每天 1440 个 ProcessInstance , api batch删除 或者ui界面上删除 都比较麻烦, 是否有自动删除 ProcessInstance 的相关功能?

**Comment by chinashenkai** [comment]
TRANSLATE:
Usage scenario: Our task scheduling is once every minute, 1440 ProcessInstance per day, api batch deletion or deletion on the UI interface is more troublesome, is there a function to automatically delete ProcessInstance?

**Comment by Tandoy** [comment]
As far as I know, there is currently no such function, such as job instances or task instances, which belong to the hot data during the activity. After the execution is over, you can write a script to archive and transfer them from the hot data storage area to the cold data storage area.
